packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -16,6 +16,11 @@ const propTypes = {
   * The text to be displayed underneath the label with the provided styling.
   */
   textValue: PropTypes.string,
+  /**
+  * Indicates if the LabelValueView is created inside a description list or not.
+  * If it is not valued, it will take false as a default, meaning that is is not a child of a description list.
+  */
+  childOfDescriptionList: PropTypes.bool,
```

Comment:
Since this is a bool, can we rename this to:
```suggestion
  isChildOfDescriptionList: PropTypes.bool,
```